src/app/plugins/checkclasses.plugin.ts

```diff
@@ -26,11 +26,15 @@ export default class CheckClassesPlugin extends Plugin {
       this.container.guildService.get(),
       userHandle
     );
-
     if (!member) {
       await message.reply('User not found.');
       return;
     }
+    const valid = Moderation.Helpers.validateUser(member.id);
+    if (!valid) {
+      await message.reply('User ID not valid.');
+      return;
+    }
```

Comment:
We have a built in validation method that you can override that gets called to see if the input is valid before the `execute` method is ran.
Look [here](https://github.com/cs-discord-at-ucf/lion/blob/eb3d25314da00e7474ddda6a7e7fae643d050576/src/app/plugins/leaderboard.plugin.ts#L16-L18) for an example.

When a message is sent, if its a command the `command handlers` calls [validate](https://github.com/cs-discord-at-ucf/lion/blob/eb3d25314da00e7474ddda6a7e7fae643d050576/src/app/handlers/command.handler.ts#L133-L136), and if that returns true, it runs execute. That way we have a standard way of validating args, and have a standardized error message.